pkg/proxy/userspace/proxier.go

```diff
@@ -687,12 +696,16 @@ func iptablesCommonPortalArgs(destIP net.IP, destPort int, protocol api.Protocol
 		args = append(args, "-d", fmt.Sprintf("%s/32", destIP.String()))
 	}
 
+	if excludeContainerSourceTraffic {
+		args = append(args, "-m", "physdev", "--physdev-is-in")
```

Comment:
Here is an example of what the code would do for external ip 104.154.64.226: 
-A KUBE-PORTALS-CONTAINER -d 104.154.64.226/32 -p tcp -m comment --comment "kube-system/mypod-svc:" -m tcp --dport 9091 -m physdev --physdev-is-in -j REDIRECT --t
o-ports 39236

I used the term "filter" in an overloaded way. It is a nat rule.